src/components/pages/paymentPage/paymentError/AlreadyPaidPage.tsx

```diff
@@ -2,13 +2,28 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import NoticeTemplate from '../../../../common/noticeTemplate/NoticeTemplate';
 
-const AlreadyPaidPage = () => {
+interface AlreadyPaidPageProps {
+  isAdditionalProduct: boolean;
+}
+
+const AlreadyPaidPage = ({ isAdditionalProduct }: AlreadyPaidPageProps) => {
   const { t } = useTranslation();
+
+  if (isAdditionalProduct) {
+    return (
+      <NoticeTemplate
+        id={'vene-already-paid-page'}
+        titleText={t('page.payment_error.already_paid.title')}
+        message={<p>{t('page.payment_error.already_paid.paid_message')}</p>}
+      />
+    );
+  }
+
```

Comment:
```suggestion
const messageKey = isAdditionalProduct
    ? 'page.payment_error.already_paid.paid_message'
    : 'page.payment_error.already_paid.paid_and_signed_message'
```

Reply:
Similar to @TatuArvela's comment :). Done.